src/GraphQL/DI/IServiceRegister.cs

```diff
@@ -1,3 +1,7 @@
+#if NET5_0_OR_GREATER
+using System.Diagnostics.CodeAnalysis;
```

Comment:
🤔 Maybe add System.Diagnostics.CodeAnalysis to `<Using Include="System.Diagnostics.CodeAnalysis" />` to dirbuildprops? I think otherwise eventually a lot of files will be augmented with that using.

Reply:
Ok I'll integrate that into my changes for this PR once I get a chance to work on it again.

Reply:
Done